swan-lake/learn-the-platform/source-code-dependencies/manage-dependencies.md

```diff
@@ -117,6 +117,10 @@ name = "mysql"
 version = "1.5.0”
 ```
 
+### Upgrading packages to use new Swan Lake Update distributions
+
+When a new Swan Lake Update distribution is released, it may include incompatible language improvements. When upgrading an existing package to use new Swan Lake Update distributions, the Ballerina compiler automatically updates the `Dependencies.toml` file with the latest compatible minor versions of the dependencies of the new distribution. This enables the existing projects to benefit from improvements in the new update releases without requiring significant modifications.
```

Comment:
```suggestion
When a new Swan Lake update distribution is released, it may include incompatible language improvements. When upgrading an existing package to use new Swan Lake update distributions, the Ballerina compiler automatically updates the `Dependencies.toml` file with the latest compatible minor versions of the dependencies of the new distribution. This enables the existing projects to benefit from improvements in the new update releases without requiring significant modifications.
```